### PR TITLE
Update gaudi.md to render on github.io

### PR DESCRIPTION
**Pull request by ctao456**
Tested all command scripts on Gaudi instance, and verified that all services are working, including both VLLM and TGI services.
Docker container printouts and yaml configs are up-to-date.

### CODE REVIEW

**Review by tomlenth — APPROVED** [review]
LGTM

File: examples/ChatQnA/deploy/gaudi.md [hunk 1]
```diff
@@ -336,18 +342,17 @@ Set the necessary environment variables to setup the use case case
...
+    export llm_service_devices=all
+
 ### Megaservice
 
     export MEGA_SERVICE_HOST_IP=${host_ip}
     export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/chatqna"
 
-### Guardrails (optional)
-
+### Gaurdrails(optional)
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:352]:
```suggestion
### Guardrails (optional)

```

File: examples/ChatQnA/deploy/gaudi.md [hunk 2]
```diff
@@ -778,30 +812,30 @@ data: b'.'
 data: b'</s>'
 data: [DONE]
 ```
-### Guardrail Microservice
-If you had enabled Guardrail microservice, access via the below curl command
 
+##### Gaurdrail Microservice
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:816]:
```suggestion
#### Guardrail Microservice

```

File: examples/ChatQnA/deploy/gaudi.md [hunk 3]
```diff
@@ -778,30 +812,30 @@ data: b'.'
 data: b'</s>'
 data: [DONE]
 ```
-### Guardrail Microservice
-If you had enabled Guardrail microservice, access via the below curl command
 
+##### Gaurdrail Microservice
+If you had enabled Gaurdrail microservice, access via the below curl command
 
-```
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:785]: These "codeblock" markers should not have been deleted.  Put the back before and after the bash command.

File: examples/ChatQnA/deploy/gaudi.md [hunk 4]
```diff
@@ -778,30 +812,30 @@ data: b'.'
...
 
+##### Gaurdrail Microservice
+If you had enabled Gaurdrail microservice, access via the below curl command
 
-```
 curl http://${host_ip}:9090/v1/guardrails\
   -X POST \
   -d '{"text":"How do you buy a tiger in the US?","parameters":{"max_new_tokens":32}}' \
   -H 'Content-Type: application/json'
-```
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:790]: Put this back

File: examples/ChatQnA/deploy/gaudi.md [hunk 5]
```diff
@@ -778,30 +812,30 @@ data: b'.'
...
 
-## Launch UI
+
+##Launch UI
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:825]:
```suggestion
## Launch UI

```

File: examples/ChatQnA/deploy/gaudi.md [hunk 6]
```diff
@@ -295,19 +301,18 @@ Set the necessary environment variables to setup the use case case
...
-### VectorDB
-
+  ### VectorDB
 
     export REDIS_URL="redis://${host_ip}:6379"
     export INDEX_NAME="rag-redis"
 
-### Embedding Service
-
+  ### Embedding Service
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:309]: No leading spaces here
```suggestion
### Embedding Service

```

File: examples/ChatQnA/deploy/gaudi.md [hunk 7]
```diff
@@ -295,19 +301,18 @@ Set the necessary environment variables to setup the use case case
     export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
     export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
 
-### VectorDB
-
+  ### VectorDB
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:304]: No leading spaces
```suggestion
### VectorDB

```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Fix guardrail misspelling and some heading issues.

**Review by preethivenkatesh — APPROVED** [review]

File: examples/ChatQnA/deploy/gaudi.md (hunk 1 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 3 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md:785]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 4 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 5 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 6 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 7 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md (hunk 2 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md]: Addressed. Thanks.

File: examples/ChatQnA/deploy/gaudi.md [hunk 8]
```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
 
 ### Dataprep
 
-    export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
-    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
-    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
+export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:300]: When you removed the four-space indent, you also changed this to no longer format it as a "code block".  You should either put these indents back, or use the explicit code-block fence using ``` before and after what should be a code-block.

File: examples/ChatQnA/deploy/gaudi.md [hunk 9]
```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
...
+export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
+export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
 
-  ### VectorDB
+### VectorDB
 
-    export REDIS_URL="redis://${host_ip}:6379"
-    export INDEX_NAME="rag-redis"
+export REDIS_URL="redis://${host_ip}:6379"
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:306]: Removing the indent lost the code-block formatting. Put the four-space indent back.

File: examples/ChatQnA/deploy/gaudi.md [hunk 10]
```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
...
 
-  ### Embedding Service
+### Embedding Service
 
-    export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
-    export EMBEDDING_SERVICE_HOST_IP=${host_ip}
-    export RETRIEVER_SERVICE_HOST_IP=${host_ip}
-    export TEI_EMBEDDING_ENDPOINT="http://${host_ip}:8090"
-    export tei_embedding_devices=all
+export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:311]: Removing the indent lost the code-block formatting. Put the four-space indent back.

File: examples/ChatQnA/deploy/gaudi.md [hunk 11]
```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
...
+export RETRIEVER_SERVICE_HOST_IP=${host_ip}
+export TEI_EMBEDDING_ENDPOINT="http://${host_ip}:8090"
+export tei_embedding_devices=all
 
 ### Reranking Service
 
-    export RERANK_MODEL_ID="BAAI/bge-reranker-base"
-    export TEI_RERANKING_ENDPOINT="http://${host_ip}:8808"
-    export RERANK_SERVICE_HOST_IP=${host_ip}
+export RERANK_MODEL_ID="BAAI/bge-reranker-base"
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:319]: Removing the indent lost the code-block formatting. Put the four-space indent back.

**Review by dbkinder — CHANGES_REQUESTED** [review]
Removing the indent lost the code-block formatting. Put the four-space indent back.

File: examples/ChatQnA/deploy/gaudi.md [hunk 12]
```diff
@@ -813,16 +813,17 @@ data: b'</s>'
 data: [DONE]
 ```
 
-##### Gaurdrail Microservice
-If you had enabled Gaurdrail microservice, access via the below curl command
+##### Guardrail Microservice
```
Comment (dbkinder) [inline examples/ChatQnA/deploy/gaudi.md:816]: This should either be an H3 (with ###) if it's at the same level as the `### Megaservice` heading above it or an H4 it it's a subheading under that Megaservice heading.  You shouldn't skip from H3 to H5 as this currently is.

**Review by dbkinder — CHANGES_REQUESTED** [review]
One more heading fix

File: examples/ChatQnA/deploy/gaudi.md (hunk 11 above)
Comment (ctao456) [reply examples/ChatQnA/deploy/gaudi.md:319]: Addressed accordingly.

**Review by dbkinder — APPROVED** [review]
LGTM, thanks

**Pull request merged**